lesson5/task2.js

```diff
@@ -0,0 +1,31 @@
+// //1
+
+// function palidromCheck(someWord) {
+//   let wordContainer = "";
+//   for (const txt of someWord) {
+//     wordContainer += txt.toLowerCase();
+//   }
+//   let reverseWord = wordContainer.split("").reverse().join("");
+//   return wordContainer === reverseWord ? "palidrom" : "nepalidrom";
```

Comment:
```suggestion
//  return wordContainer.split("").reverse().join("");
```
Функция должна возвращать булевое значение